float32 motor_angle
float32 motor_v
float32 current
float32 voltage